plugins/builder/osbuild.py

```diff
@@ -203,6 +213,18 @@ def compose_status(self, compose_id: str):
 
         return ComposeStatus.from_dict(res.json())
 
+    def compose_logs(self, compose_id: str):
+        url = urllib.parse.urljoin(self.url, f"compose/{compose_id}/logs")
+
+        res = self.http.get(url)
+
+        if res.status_code != 200:
+            body = res.content.decode("utf-8").strip()
+            msg = f"Failed to get the compose logs: {body}"
+            raise koji.GenericError(msg) from None
+
+        return ComposeLogs.from_dict(res.json())
```

Comment:
Out of curiosity: Is a classmethod conventional in situations like this? I'd have written something like
```python
ComposeLogs(**res.json())
```

Reply:
Good question, I *felt* that was a bit more explicit and supports the more nested case better, but I think it is not clear cut.